db/migrate/20230523134506_claim_details.rb

```diff
@@ -7,5 +7,8 @@ def change
     add_column :claims, :preparation_time, :string
     add_column :claims, :time_spent_hours, :integer
     add_column :claims, :time_spent_mins, :integer
+    add_column :claims, :other_info, :string
+    add_column :claims, :conclusion, :string
+    add_column :claims, :concluded, :string
```

Comment:
other_info and conclusion should be text instead of string fields as they are text-boxes in the UI so likely to be longer than sting can accommodate.

Reply:
cheers - string goes to 255 chars text to 30000 chars

Reply:
database defintion changed to text but form doesnt seem to recognised :text - saved a 10k string  in database